src/helpers/googleFont.ts

```diff
@@ -27,6 +27,7 @@ export const webFontLoader = () => {
   document.getElementsByTagName('head')[0].appendChild(script);
 };
 
+// choose new icons here https://fonts.google.com/icons?icon.set=Material+Icons
```

Comment:
Maybe we could this link to the readme as well?

Reply:
added